packages/altive_lints/pubspec.yaml

```diff
@@ -15,6 +15,6 @@ environment:
   sdk: ^3.0.0
 
 dependencies:
-  analyzer: ^6.5.0
+  analyzer: ^6.7.0
```

Comment:
The `analyzer` version has been updated to align with the current Flutter Stable.